src/_P036_FrameOLED.ino

```diff
@@ -414,8 +417,8 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
         {
           /* *INDENT-OFF* */
           F("SSID"),     F("SysName"),      F("IP"),   F("MAC"),         F("RSSI"),
-          F("BSSID"),    F("WiFi channel"), F("Unit"), F("SysLoad"),     F("SysHeap"),
-          F("SysStack"), F("Date"),         F("Time"), F("PageNumbers"),
+          F("BSSID"),          F("WiFi channel"),          F("Unit"),                  F("SysLoad"),                F("SysHeap"),
+          F("SysStack"),       F("Date"),                  F("Time"),                  F("PageNumbers"),
```

Comment:
Hm, your formatter seems to ignore the directive to not touch the indentation...